pickler/summarize.py

```diff
@@ -767,42 +847,44 @@ def summarize(j, lariatcache):
                 else:
                     summaryDict['mpirx'] = { 'error': 2, 'error_msg': 'missing ib_sw or lnet data points' }
 
+
+
         del totals['cpu']
 
-        converttooutput(series, summaryDict, j)
-        converttooutput(totals, summaryDict, j)
+        converttooutput(series, summaryDict, job)
+        converttooutput(totals, summaryDict, job)
 
         summaryDict['analysis'] = {}
-        summaryDict['analysis']['catastrophe'] = compute_catastrophe(j)
+        summaryDict['analysis']['catastrophe'] = compute_catastrophe(job)
 
     # add in lariat data
     if lariatcache != None:
-        lariatdata = lariatcache.find(j.id, j.acct['start_time'], j.acct['end_time'])
+        lariatdata = lariatcache.find(job.id, job.acct['start_time'], job.acct['end_time'])
         if lariatdata != None:
             summaryDict['lariat'] = lariatdata
         else:
             summaryDict['Error'].append("Lariat data not found")
 
     # add hosts
     summaryDict['hosts'] = []
-    for i in j.hosts.keys():
-        summaryDict['hosts'].append(i)
+    for i in job.hosts.keys():
+        summaryDict['hosts'].append(i.encode('ascii', 'ignore') if type(i) == unicode else i)
 
     summaryDict['collection_sw'] = "tacc_stats " + " ".join(tacc_version)
 
     # add account info from slurm accounting files
-    summaryDict['acct'] = j.acct
+    summaryDict['acct'] = fix_unicode(job.acct)
```

Comment:
Please explain why fix_unicode is done here.

Reply:
This has been removed.